pkg/store/bucket.go

```diff
@@ -1102,13 +1119,25 @@ func (s *BucketStore) LabelNames(ctx context.Context, req *storepb.LabelNamesReq
 	if err := g.Wait(); err != nil {
 		return nil, status.Error(codes.Internal, err.Error())
 	}
+
+	var anyHints *types.Any
+	var err error
+
+	if anyHints, err = types.MarshalAny(resHints); err != nil {
+		err = status.Error(codes.Unknown, errors.Wrap(err, "marshal label names response hints").Error())
+		return nil, err
+	}
```

Comment:
Why declaring `anyHints` and err in separate lines? Why not:

```suggestion
	anyHints, err := types.MarshalAny(resHints); 
	if err != nil {
		return, status.Error(codes.Unknown, errors.Wrap(err, "marshal label names response hints").Error())
	}
```